inc/template-tags.php

```diff
@@ -37,48 +37,32 @@ function nightingale_wp_entry_footer() {
 
 	?>
 	<div class="o-layout">
-		<div class="o-layout__item u-9/12@lg">
+		<div class="o-layout__item u-6/12@lg">
```

Comment:
Is this being used in more than the home page secondary? Does this have a knock on effect for post listings?

Reply:
Yes, it's used on all posts but there's no knock-on effect as there's still room for a comment count on the left and "read more" on the right.